core/src/main/java/org/commonjava/maven/ext/core/ManipulationManager.java

```diff
@@ -80,7 +80,7 @@
 
     public static final String MARKER_FILE = MARKER_PATH + File.separatorChar + "pom-manip-ext-marker.txt";
 
-    private static final String REPORT_JSON_DEFAULT = File.separatorChar + "manipulation.json";
+    public static final String REPORT_JSON_DEFAULT = "alignmentReport.json";
```

Comment:
Made this constant public to reuse (multiple times) from test files.

Reply:
Ok

Reply:
Also, you can see I removed the `File.separatorChar`. I am not sure having it here was intentional, as it adds another `File.separator` later on anyway

Reply:
Ok, np.